@navikt/ds-css/popover.css

```diff
@@ -4,6 +4,7 @@
   box-shadow: 0 0.05rem 0.25rem 0.125rem rgba(0, 0, 0, 0.08);
   border: var(--navds-border-default);
   border-radius: 2px;
+  padding: 0.5rem;
```

Comment:
Jeg tenker det er best å la padding være opp til innholdet. Da slipper man f.eks. å måtte overskrive padding for å lage en popover-meny der menypunktene skal gå helt ut til kanten. Jeg ser at både popper.js og material-ui overlater padding til innholdet.

Reply:
Ser ut som det er en bug at innholdet ikke aligner med arrow som vist i screenshot ovenfor. Hvis vi fikser den så er jeg enig i at padding ikke trengs. Har jaktet etter buggen en stund nå, men finner ikke noe konkret. Antar det har noe med kalkulering av padding eller hvordan `Popperjs` oppdaterer.

Reply:
Fant buggen: ref={arrowRef} ble ikke oppdatert riktig. Ved å bare bruke `data-popper-arrow` istedenfor en ref så ble innholdet alignet med arrow 
<img width="164" alt="Screenshot 2021-04-07 at 11 23 58" src="https://user-images.githubusercontent.com/26967723/113843339-c968b000-9793-11eb-8a9a-eaf84b12b459.png">